Library/Homebrew/test/utils/popen_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "utils/popen"
+
+describe Utils do
+  describe "::popen_read" do
+    it "reads the standard output of a given command" do
+      expect(subject.popen_read("sh", "-c", "echo success").chomp).to eq("success")
```

Comment:
This isn't ideal:
<img width="723" alt="screen shot 2017-02-20 at 10 59 22" src="https://cloud.githubusercontent.com/assets/125011/23122318/a60c3e10-f75b-11e6-96d8-ce01fd7c6436.png">

We do generally prefer <= 80 characters